WcaOnRails/config/locales/ru.yml

```diff
@@ -1402,11 +1403,11 @@ ru:
     nearby_competitions:
       #original_hash: 241801a
       competitions:
-        other: '%{count} WTF'
-        many: '%{count} соревнований'
-        few: '%{count} соревнования'
-        one: '%{count} соревнование'
         zero: Нет соревнований
+        one: '%{count} соревнование'
+        few: '%{count} соревнования'
+        many: '%{count} соревнований'
+        other: '%{count} WTF'
```

Comment:
Strange. So is this a bug with @jonatanklosko's app that it's requiring you to enter these keys, even though they're not used? Or is it a bug with our local testing code?

Either way, this seems good to merge to me.